### PR TITLE
Api update

### PR DESCRIPTION
**Pull request by Eyiowuawi**
Good evening sir,
Kindly help me merge this branch

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (mayowadabiri) [inline .vscode/settings.json:1]: Can you ignore this file

File: src/App.js [hunk 2]
```diff
@@ -21,10 +22,36 @@ import Bookmarks from 'pages/Bookmarks/Bookmarks';
...
+      console.log(loggedUser.email)
+    } catch (error) {
+      console.log(error)
+      toast.error('Unable to fetch user details');
+      
+    }
+  };
+
+  useEffect(() => {
+    if (isLoggedIn === true) {
```
Comment (mayowadabiri) [inline src/App.js:51]: Change it to `if (isLoggedIn){`

File: src/App.js [hunk 3]
```diff
@@ -21,10 +22,36 @@ import Bookmarks from 'pages/Bookmarks/Bookmarks';
...
 import 'aos/dist/aos.css';
+import { useBookmarkState } from 'context/context';
+import { getUserDetails } from 'services/auth'
+import { toast } from 'react-toastify';
+
 
 function App() {
   AOS.init();
   const location = useLocation();
+  const { isLoggedIn, setUser, user } = useBookmarkState();
```
Comment (mayowadabiri) [inline src/App.js:33]: By the way, `useBookmarkState` is not a viable name for this.

File: src/components/Nav/Nav.jsx [hunk 4]
```diff
@@ -7,22 +7,27 @@ import {
 } from 'react-icons/md';
 import Profile from 'assets/profile.svg';
 import { useState } from 'react';
+import { logout } from 'services/auth';
+import { toast } from 'react-toastify';
 
 const Nav = ({ toggle, handleToggle }) => {
-  const { isLoggedIn, setIsLoggedIn } = useBookmarkState();
-
+  const { isLoggedIn, setIsLoggedIn, user } = useBookmarkState();
```
Comment (mayowadabiri) [inline src/components/Nav/Nav.jsx:14]: you can destructure only you want you need from the `useBookmarkState` pls.. if you don't need `isLoggedIn` remove it

File: src/context/context.jsx [hunk 5]
```diff
@@ -6,12 +6,13 @@ const UserContext = createContext();
 
 const Context = ({ children }) => {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [user, setUser] = useState('');
```
Comment (mayowadabiri) [inline src/context/context.jsx:9]: the default state should be `null` not a string

File: src/services/auth.jsx [hunk 6]
```diff
@@ -17,15 +20,42 @@ export const forgetPassword = async (payload) => {
...
+};
+
+//Update user profile API request
+export const updateProfile = async (payload, id) => {
+  const response = baseUrl.patch(`auth/user/edit-profile/:_${id}`, payload);
+  return response;
+};
+//Post a house API request
+export const post_House = async (payload, id) => {
+  const response = baseUrl.post(`/addHouse?_id=${id}`, payload);
```
Comment (mayowadabiri) [inline src/services/auth.jsx:44]: From the endpoint, you don't need to pass an id param

**Review by mayowadabiri — CHANGES_REQUESTED** [review]

**Comment by mayowadabiri** [comment]
Also, let's remove all console.logs

File: .vscode/settings.json (hunk 1 above)
Comment (Eyiowuawi) [reply .vscode/settings.json:1]: ok sir

File: src/App.js (hunk 2 above)
Comment (Eyiowuawi) [reply src/App.js:51]: correction implemented

File: src/App.js (hunk 3 above)
Comment (Eyiowuawi) [reply src/App.js:33]: I've changed it to useContextState

File: src/components/Nav/Nav.jsx (hunk 4 above)
Comment (Eyiowuawi) [reply src/components/Nav/Nav.jsx:14]: isLoggedIn, setIsLoggedIn and user is needed that is why I destructured them, sir

File: src/context/context.jsx (hunk 5 above)
Comment (Eyiowuawi) [reply src/context/context.jsx:9]: correction effected Sir

File: src/services/auth.jsx (hunk 6 above)
Comment (Eyiowuawi) [reply src/services/auth.jsx:44]: params is needed for the endpoint sir. there is a need to track a due based on the person that uploaded the house

**Comment by Eyiowuawi** [comment]
I have implemented all corrections @mayowadabiri... if there is any other thing I need to effect before you merge kindly let me know sir.

**Comment by Eyiowuawi** [comment]
the corrections have been committed and pushed as well

**Comment by Eyiowuawi** [comment]
All console.log has been removed. The code is ready to be merged sir.

**Pull request merged**